test/unit/math/mix/prob/frechet_test.cpp

```diff
@@ -0,0 +1,11 @@
+#include <stan/math/mix.hpp>
+#include <test/unit/math/test_ad.hpp>
+#include <gtest/gtest.h>
+
+TEST(mathMixScalFun, frechet_lpmf_derivatives) {
+  auto f = [](const auto& y, const auto& alpha, const auto& beta) {
+    return stan::math::frechet_lpdf<false>(y, alpha, beta);
+  };
+
+  stan::test::expect_ad(f, 2.0, 1.0, 1.0);
```

Comment:
I added this test since the higher order Frechet stuff was failing. I think I could remove it but it seems fine to me.